app/services/session.js

```diff
@@ -109,6 +110,7 @@ export default class Auth0Service extends Service {
         }
       }
     } else {
+      // QUESTION: should this be setting this.isAuthenticated to false too?
```

Comment:
![Screen Shot 2022-08-14 at 6 57 38 PM](https://user-images.githubusercontent.com/19944631/184547162-0d037cf1-da15-4df6-ae93-d6723572ea4a.png)
This is the only place where it is consumed, `getAccessToken` this function is only responsible for giving token. So this should be fine.